modules/mediawiki/files/robots.txt

```diff
@@ -1,9 +1,13 @@
 # Disallow API and special pages
 User-Agent: *
-Disallow: /w/api.php
-Disallow: /w/index.php?title=Special:
+Allow: /w/load.php?
+Disallow: /w/
 Disallow: /wiki/Special:
 
 # Throttle YandexBot
 User-Agent: YandexBot
 Crawl-Delay: 2.5
+
+# Allow the Internet Archiver to index action=raw and thereby store the raw wikitext of pages
+User-agent: ia_archiver
+Allow: /*&action=raw
```

Comment:
Is &action=raw ever used by IA?

Reply:
Yes, under Miraheze's current robots.txt directives, IA is able to archive raw pages. I just saved the main page with &action=raw. [Link](https://web.archive.org/web/20180224181708/https://meta.miraheze.org/wiki/Miraheze?action=raw).